Gemfile

```diff
@@ -31,6 +31,7 @@ gem "faker", "~> 2.14.0"
 gem "sidekiq"
 
 gem "doorkeeper", "< 5.6"
+gem "nobspw"
```

Comment:
Why adding nobspw as gem? I've seen some discussions inside the decidim community and they have removed it.

https://github.com/decidim/decidim/issues/8516
https://github.com/decidim/decidim/pull/8455

Reply:
I did it because it was necessary for the `nobspw.rb` initializer. I will delete this initializer instead